settings.gradle

```diff
@@ -1,3 +1,13 @@
-rootProject.name = 'wrims'
+rootProject.name = 'cvwrims'
```

Comment:
Why did you have to change the rootProject.name?
If the 'cv' of 'cvwrims' is to reflect the org name, then this should be renamed to 'cvm-wrims'

Reply:
Since the "wrims-plugin" module was renamed to "wrims", it felt confusing to have the root project also named 'wrims'. I can either set root project to "cvm-wrims" or change it back to "wrims" so long as it doesn't cause any build conflicts. .